verifier.py

```diff
@@ -25,7 +28,7 @@
 
 
 # Takes a URL and produces its relative file name.
-def get_path_from_url(self, url):
+def get_path_from_url(url):
     # Remove http://domain
```

Comment:
This method, along with "generate_page_dictionary()", could both become private instance methods of the class.